src/mmw/js/src/compare/templates/compareModificationsPopover.html

```diff
@@ -0,0 +1,23 @@
+{% macro modificationsTable(caption, modifications) %}
```

Comment:
Same! I stole this from the original implementation of the mod table. Nunjunks has a quite a few tricks up its sleeves